### PR TITLE
chore(xo): setup linter

### PR DESCRIPTION
**Pull request by mastilver**
Once v1.3 is release, and #58 is merged; this should be merged with linting fixes

Any additional rules, you want in / disable?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,5 +51,18 @@
...
       "spec"
     ]
+  },
+  "xo": {
+    "space": 2,
+    "rules": {
+      "space-before-function-paren": ["error", "never"]
+    },
+    "globals": [
+      "describe",
```
Comment (D34THWINGS) [inline package.json]: You can use:
```json
{
  "env": {
    "jasmine": true
  }
}
```
instead of adding every global one by one

Comment (mastilver) [reply package.json]: Damn, didn't know about this... Super useful 👍

Comment (D34THWINGS) [reply package.json]: FYI https://eslint.org/docs/user-guide/configuring#specifying-environments

**Comment by shellscape** [comment]
I'm going to be taking this an ESLint route. Thanks for the work on this PR previously.

**Pull request closed without merging**